tcs-service/pom.xml

```diff
@@ -12,7 +12,7 @@
 
   <groupId>com.transformuk.hee.tis</groupId>
   <artifactId>tcs-service</artifactId>
-  <version>6.48.1</version>
+  <version>6.49.0</version>
```

Comment:
Just checking, is it intentional that the change to tcs-client is considered a patch version and this change is considered minor?

Reply:
I think tcs-client is a minor too.